src/modules/classRecord.js

```diff
@@ -1,4 +1,5 @@
 /* eslint-disable import/prefer-default-export */
```

Comment:
- [ ] As I pointed out above, As required by Microverse and Following the best practices, you are not meant to use any linters diable rule in your codebase. Linters are there to make sure your code follows the best standard, I'll suggest you remove this and fix the error. :+1: